webservice/src/test/java/pl/edu/pjatk/lnpayments/webservice/auth/resource/AuthResourceIntegrationTest.java

```diff
@@ -57,11 +65,26 @@ void shouldReturn409WhenEmailIsTaken() throws Exception {
     }
 
     @Test
-    void shouldReturn400ForInvalidParams() throws Exception {
+    void registrationShouldReturn400ForInvalidParams() throws Exception {
         RegisterRequest request = new RegisterRequest("test@test.pl", null, "");
         mockMvc.perform(post("/auth/register")
                         .content(new ObjectMapper().writeValueAsString(request))
                         .contentType(MediaType.APPLICATION_JSON))
                 .andExpect(status().isBadRequest());
     }
+
+    @Test
+    void renewalShouldReturnOkAndNewToken() throws Exception {
```

Comment:
Fajnie by było dodać testy, kiedy coś się nie powiedzie np.: niepoprawny token. Nie wiem jak to jest teraz obsługiwane, może trzeba wywołać isTokenValid, przed wygenerowaniem nowego.